espsim/electrostatics.py

```diff
@@ -92,8 +99,6 @@ def GetEspSim(prbMol,
               basisPsi4 = '3-21G',
               methodPsi4 = 'scf',
               gridPsi4 = 1,
-              nocheck=False,
```

Comment:
It seems like your PR was based on an old espsim version, missing several recent commits (e.g. the no check option, seeds for the embedding routines, etc) - could you rebase to the most up-to-date version?